jekyll/_cci2/configuration-reference.md

```diff
@@ -539,6 +539,51 @@ The directory layout should match the [classic CircleCI test metadata directory
     path: /tmp/test-results
 ```
 
+#### **`persist_to_workspace`**
+
+Special step used to persist a temporary file to be used by another job in the workflow. 
+
+Key | Required | Type | Description
+----|-----------|------|------------
+paths | Y | String | Directory containing temporary files
+{: class="table table-striped"}
+
+##### _Example_
+
+``` YAML
+- persist_to_workspace:
+    paths: /tmp/file-name
+```
+
+#### **`attach__workspace`**
+
+Special step used to attach a temporary file generated by another job in the workflow. 
+
+Key | Required | Type | Description
+----|-----------|------|------------
+at | Y | String | Directory containing temporary files
+{: class="table table-striped"}
+
+##### _Example_
+
+``` YAML
+- attach_workspace:
+    at: /tmp/file-name
+```
+
+A job has one implicit workspace when it runs in a workflow (although using it is optional). In the fan-in case, workspaces of upstream jobs are merged. **Note**: Name conflicts will cause error currently. When a job finishes, its workspace is frozen and cannot be changed. The same workspace is used if a downstream job is rerun. Consider a workflow of J1 -> J2 -> J3
+Any rerun of J2 will see the same workspace as the original run. Use workspaces to pass along data unique to this run and which are needed for downstream jobs.
+
+Note the following distinctions between Artifacts, Workspaces, and Caches:
+
+| Type      | lifetime        | Use                      | Example |
+|-----------|-----------------|------------------------------------|---------
+| Artifacts | Months          | Preserve long-term artifacts. |  Available in the Artifacts tab of the Build details under the `tmp/circle-artifacts.<hash>/container`   or similar directory.     |
+| Workspaces | Duration of workflow        | Move temporary files around for use by jobs run later in the workflow. | The `persist_to_workspace: <directory>` key under jobs, refer to the file with `attach_workspace: at: <directory>`.    |
+| Caches    | Months          | Store dependency or source code data in fast memory for quick retrieval.          |  The `save_cache` job step with a `path` to a list of directories to add and a `key` to uniquely identify the cache (for example, the branch, build number, or revision).   Restore the cache with `restore_cache` and the appropriate `key`. |
```

Comment:
"fast memory" ... "quick retrieval" is not correct.  Caches are not faster than workspaces.